nidirect_news/tests/src/Nightwatch/Tests/featuredContentListViewSwitch.js

```diff
@@ -0,0 +1,37 @@
+let newsContentTitles = [];
+let homepageFeaturedTitles = [];
+
+module.exports = {
+  '@tags': ['regression'],
```

Comment:
Felt like a useful, general tag to keep to ensure we can easily check for all previously detected regressions.